R/helpers.R

```diff
@@ -85,3 +85,15 @@ replaceDupeMeasureNames = function(measures, x = "id") {
   meas.names[dupes] = new.names
   unlist(meas.names)
 }
+
+# checks for duplicated entries in learner.names and stops
+# with error message containing the learner name that appeared more
+# than once
+checkDuplicatedLearnerNames = function(learner.names) {
+  dupl = duplicated(learner.names)
+  if (any(dupl)) {
+    dupl.learner = learner.names[which.first(dupl)]
```

Comment:
what if multiple names are duplicated? the error will just return one of the pairs. what about `unique(learner.names[dupl]))`.